core/pom.xml

```diff
@@ -436,6 +436,21 @@ THE SOFTWARE.
       <groupId>org.codehaus.groovy</groupId>
       <artifactId>groovy-all</artifactId>
       <version>${groovy.version}</version>
+      <type>pom</type>
```

Comment:
This is a bad idea. We include JARs here

Reply:
I agree. Using the POM / BOM approach would be a good idea if this was inside of `dependencyManagement` but it isn't.